data/hai/hai outfits.txt

```diff
@@ -108,6 +133,66 @@ effect "ion impact"
 	"velocity scale" -.05
 
 
+outfit "Ionic Blaster Prototype"
+	category "Guns"
+	cost 530000
+	thumbnail "outfit/hai ionic blaster"
+	"mass" 19
+	"outfit space" -19
+	"weapon capacity" -19
+	"gun ports" -1
+	weapon
+		sprite "projectile/ion bolt"
+			"frame rate" 5
+		sound "ion blast"
+		"hit effect" "ion impact"
+		"inaccuracy" 4
+		"velocity" 18
+		"lifetime" 30
+		"reload" 30
+		"firing energy" 72
+		"firing force" 12
+		"firing heat" 108
+		"hit force" 72
+		"shield damage" 70
+		"hull damage" 30
+		"ion damage" 1
+	description "Since the beginning of their conflict with the rest of the Hai, the Unfettered have attempted to gain a technological edge despite their limited resources. To that end, they began developing the Ionic Blaster. Much smaller than the venerable Ion Cannon, but still able to apply the same weapon jamming and energy draining ion effects, albeit to a much lesser extent. It also has enough raw damage to be useful in all situations."
```

Comment:
```suggestion
	description "Since the beginning of their conflict with the rest of the Hai, the Unfettered have attempted to gain a technological edge despite their limited resources. To that end, they began developing the Ionic Blaster. It is much smaller than the venerable Ion Cannon, but still able to apply the same weapon jamming and energy draining ion effects, albeit to a much lesser extent. It also has enough raw damage to be useful in all situations."
```

Reply:
I think it'd be better to replace the full stop at the end of this sentence to a comma and combine it with the next one rather than having both begin with "it."

Reply:
Is this what you meant?

Reply:
```suggestion
	description "Since the beginning of their conflict with the rest of the Hai, the Unfettered have attempted to gain a technological edge despite their limited resources. To that end, they began developing the Ionic Blaster. Much smaller than the venerable Ion Cannon but still able to apply the same weapon jamming and energy draining effects, albeit to a much lesser extent, it also has enough raw damage to be useful in all situations."
```

@Hurleveur , I thought this was what @warp-core meant?

Breaks down to "much smaller... it also has enough..."

This way also needs a comma removed and the second instance of ion becomes more redundant in the pacing change.

Reply:
I applied that change